packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx

```diff
@@ -140,11 +152,11 @@ export function useEarningsBreakdown(isGrouped: boolean) {
               : `Leveraged ${underlying.symbol} Lend`,
           caption: formatCaption(asset, debt) || '',
         },
-        incentivesEarnings: totalIncentiveEarnings
-          ? totalIncentiveEarnings
-              .toFiat(baseCurrency)
-              .toDisplayStringWithSymbol(2, true, false, 'en-US', true)
-          : '0',
+        incentivesEarnings:
+          totalIncentiveEarnings &&
+          totalIncentiveEarnings.toFloat() > minEarningsValue
+            ? formatCryptoWithFiat(baseCurrency, totalIncentiveEarnings)
+            : '-',
```

Comment:
I thought we were going to just show a zero in these cases? I don't think we should be doing this minEarningsValue comparison anywhere